lib/compat/wordpress-6.1/class-wp-theme-json-6-1.php

```diff
@@ -812,33 +816,45 @@ function( $pseudo_selector ) use ( $selector ) {
 			$block_rules .= '}';
 		}
 
-		if ( static::ROOT_BLOCK_SELECTOR === $selector ) {
+		return $block_rules;
+	}
 
-			if ( $use_root_padding ) {
-				$block_rules .= '.wp-site-blocks { padding-top: var(--wp--style--root--padding-top); padding-bottom: var(--wp--style--root--padding-bottom); }';
-				$block_rules .= '.has-global-padding { padding-right: var(--wp--style--root--padding-right); padding-left: var(--wp--style--root--padding-left); }';
-				$block_rules .= '.has-global-padding > .alignfull { margin-right: calc(var(--wp--style--root--padding-right) * -1); margin-left: calc(var(--wp--style--root--padding-left) * -1); }';
-				$block_rules .= '.has-global-padding > .alignfull > :where([class*="wp-block-"]:not(.alignfull):not(.alignfull):not([class*="__"]),p,h1,h2,h3,h4,h5,h6,ul,ol) { padding-right: var(--wp--style--root--padding-right); padding-left: var(--wp--style--root--padding-left); }';
-			}
+	/**
+	 * Outputs the additional CSS for layout rules on the root.
+	 * This needs to be separate from get_styles_for_block,
+	 * so that it can load later and override block rules.
+	 *
+	 * @param string $selector The root node selector.
+	 * @param array $block_metadata The metadata for the root block.
+	 * @return string The additional root rules CSS.
+	 */
+	private function get_additional_root_rules( $selector, $block_metadata ) {
+		$css = '';
+
+		if ( $use_root_padding ) {
```

Comment:
`$use_root_padding` is not yet defined in this function, so this is throwing a warning (and these styles aren't output when `"useRootPaddingAwareAlignments"` is set to true in `theme.json`.